src/User.php

```diff
@@ -60,7 +60,7 @@ private function isIncluded(Experiment $experiment): bool
             }
         }
 
-        if ($experiment->url && !Util::urlIsValid($experiment->url)) {
+        if ($experiment->url && !Util::urlIsValid($experiment->url, $this->client->config->url)) {
```

Comment:
`$this->client->config->url` may be null, but `urlIsValid` is always expecting a string.  Can you either make sure this argument is always a string or change the function signature to accept nulls?